metadata.go

```diff
@@ -52,22 +52,22 @@ func (im *InstanceMetadata) parse() error {
 			log.Errorf("Error unmarshalling: %s", err.Error())
 			return fmt.Errorf("error unmarshalling: %s", err.Error())
 		}
-		im.parsed = parsedDoc["d"].(map[string]interface{})
+		im.Parsed = parsedDoc["d"].(map[string]interface{})
 	}
 	return nil
 }
 
 // GetMap returns a map of the metadata parameters for this instance
 func (im *InstanceMetadata) GetMap() map[string]interface{} {
```

Comment:
Now that the underlying field is exported, this method isn't necessary, but I doubt we can get away with removing it, as that would break existing callers.